src/scheduler/globals.cpp

```diff
@@ -43,6 +43,9 @@
 #include <pthread.h>
 #include <limits.h>
 
+#include <unordered_map>
+#include <string>
+
```

Comment:
No real need to include these, you included them in globals.h

Reply:
I disagree with this, you should keep them here since you're using them in this file. If it gets removed from globals.h, they'll have to just add them here too.

Reply:
@bhroam do you feel strongly about not including them?

Reply:
I still think having them in the .h file is enough.  It is a file specific .h file, so it has to be included in globals.cpp.  The only way we can remove it from the .h file is to remove the variables.  If we do that, we'll be removing them from the .cpp file as well.

Reply:
You removed your unordered_map.  No need for the header any more.